packages/wiki-docs/Contributing.md

```diff
@@ -203,4 +203,104 @@ npx @seasketch/geoprocessing@0.15.1-beta.1 init 0.15.1-beta.1
+
+OPTION 1 - packages/i18n
+
+- publish it as @seasketch/geoprocessing-i18n
+- translations are stored within the published package
+- on project init, translations are copied from i18n directory to project space
+- project report client imports i18n.ts file from i18n lib
+- geoprocessing and template packages can import the sibling i18n package and import i18n.ts the same way.
+
+pros
+
+- sibling packages import from @seasketch/geoprocessing-i18n
+- i18n package is independent of geoprocessing package on npm and could potentially be upgraded independently by a project.
+- translation:upgrade - bump the i18n package version and `translation:upgrade` bin script in project space will merge the updates.  A function could be provided within i18n
+
+cons
+
+- have to transpile and publish a separate package from geoprocessing.  But that opens the door to breaking down the gp monolith.  Opportunity to use esm module?
+
+OPTION 2 - geoprocessing/src/i18n
+
+- store translations and related code in geoprocessing package
+- monorepo root calls 
+- bundle lang json files just like templates into build
+- projects and sibling template packages will import i18n init from @seasketch/geoprocessing/i18n entry point (similar to dataproviders)
+on project init, generate i18n directory in project space, copy the lang assets out.
+- each project/template will have its own translate CLI commands.
+
+## Creating core translations
+
+
+babel - with babelrc, uses i18n-extract to bundle translations, which can be served to the client.
+
+Namespaces is `core` and combines:
+- geoprocessing (src/components, src/rbcs/components, src/iucn/)
+- template-blank-project
+- template-ocean-eez
+- template-addon-* (future packages)
+
+## Creating gp project translations
+
+
+
+## Loading project translations
+
+i18n.ts is used to detect the current language and load the translations from teh backend
+
+Load core translations
+Load project translations
+Merge them?
+
+## Questions
+
+- Should we use SeaSketch Next project for gp and gp project translations?  Yes I think so in order to have centrally managed translator access
+
+Yes
+
+- Should translation machinery be kept in a monorepo package that works across all packages to pull together and merge translations?
+
+- how does translation loading work on client side?
+
+load should happen in top level of report client, so that it can load project specific translations, which may override core translations.
+
+Report developer should not have to deal with this.  Perhaps move ReportPage out of client-ui as   Should be built-in or part of template-blank-project report client, and create:function client.
+
+Answers
+
+- what to do with big chunks of text with html sprinkled in? such as used in learn more section of reports?
+  - it looks like you can wrap text that includes html tags into <Trans/> component
```

Comment:
I've been able to do this in SSN, but limit it to inline tags like anchor, code, em, etc.